tests/unit/s2n_handshake_test.c

```diff
@@ -434,6 +434,30 @@ int main(int argc, char **argv)
         }
     }
 
+    /* Ensure that a handshake can be performed after all non-standard file descriptors are closed */
+    {
+        long max_file_descriptors = sysconf(_SC_OPEN_MAX);
+        EXPECT_SUCCESS(max_file_descriptors);
+        EXPECT_TRUE(max_file_descriptors < INT_MAX);
```

Comment:
Can max_file_descriptors not be INT_MAX? Or theoretically INT_MAX+1, since a fd is an int and can be any value from 0 up to and including INT_MAX? Or is there a rule limiting the values somewhere?

Reply:
Yes true. I fixed the logic here to check <= INT_MAX, and have the loop iterate from 0 to <= max_file_descriptors.